Game/Game.cs

```diff
@@ -18,7 +18,7 @@ public class Game : Node
 	public static bool WorldOpen = false;
 	public static StructureRootClass StructureRoot;
 
-	public static float MouseSensitivity = 1;
+	public static float MouseSensitivity = 3;
```

Comment:
This should be renamed to LookSensitivity as it is not mouse nor joystick specific

Can we increase this to a higher value and compensate even more by dividing by a higher value in the mouse motion even? Looking around with a stick is *painfully* slow on my SC (could be an issue with my hardware, correct me if this is so)